x-pack/test/functional/apps/ml/anomaly_detection/saved_search_job.ts

```diff
@@ -77,7 +77,7 @@ export default function ({ getService }: FtrProviderContext) {
           frontCardTitle: 'AAL',
         },
         row: {
-          recordCount: '34,416',
+          recordCount: '86,274',
```

Comment:
Not sure if this change was intended. But it's failing on my local deployment and on CI. I think we should revert this.

Reply:
Reverted here 958b660